client/src/entrypoints/admin/core.js

```diff
@@ -308,11 +308,8 @@ if (!wagtail.ui) {
 const DROPDOWN_SELECTOR = '[data-dropdown]';
 const LISTING_TITLE_SELECTOR = '[data-listing-page-title]';
 const LISTING_ACTIVE_CLASS = 'listing__item--active';
-const ICON_DOWN = 'icon-arrow-down';
-const ICON_UP = 'icon-arrow-up';
 const IS_OPEN = 'is-open';
```

Comment:
This was all dead code. Dropdown up/down chevrons had already been rewritten to use the `is-open` class only to determine which icon to show.